native_client/beam_search.h

```diff
@@ -189,4 +189,4 @@ class KenLMBeamScorer : public tensorflow::ctc::BaseBeamScorer<KenLMBeamState> {
   }
 };
 
-#endif /* BEAM_SEARCH_H */
\ No newline at end of file
+#endif /* BEAM_SEARCH_H */
```

Comment:
nit: blank change because of line ending

Reply:
That was on purpose to silence a warning about no line end at the end of the file